### ISSUE TITLE
Mac deprecated version unable to make new characters

### ISSUE THREAD

**Issue by JoshuaNotowitz** [pyfa-org/Pyfa#660]
the character editor works but when i try to add a new character the confirm button for the char doesn't work and i cant get past this step when creating a character. the same issue occurs when making a copy of a char.

**Comment by blitzmann** [comment]
Please send me a copy of `~/.pyfa/saveddata.db` at sable.blitzmann@gmail.com. Another user has a similar issue and it was database corruption

**Comment by blitzmann** [comment]
Also a note: I tested with the latest release, and experienced the same issue. Testing the latest development code would not reproduce. I'm going to test a packaged version of the latest code and see if that does anything.

**Comment by blitzmann** [comment]
update: I can also reproduce, with a fresh database as well. This happens:

```
2016-06-23 21:46:28,129 gui.contextMenu          DEBUG    1 new IDs created for this menu
Traceback (most recent call last):
  File "/Users/ryan/Desktop/pyfa.app/Contents/Resources/gui/characterEditor.py", line 47, in Validate
    elif text in [x.name for x in profileEditor.entityEditor.choices]:
AttributeError: 'CharacterEntityEditor' object has no attribute 'entityEditor'
```

I am not sure whats going on here, and debugging it in a packaged pyfa on osx is going to be a bitch. hopefully I'll have a fix by the release next week.

**Comment by blitzmann** [comment]
Someone contacted me about the same issue on Linux Mint. They were running wxPython 2.8, which is the version the deprecated OS X version runs out - I did not test running pyfa through an interpreter using wxPython 2.8. It may even break in Windows when using this version, which would help debugging efforts.

**Comment by blitzmann** [comment]
This should be fixed with b4bd6e7. It was ultimately an issue with wxPython 2.8 that was seemingly fixed in 3.0, but I was able to get it to work with both versions. I'll be making a release sometime this week and then closing this issue - if you notice that's it's still broken at that time, let me know. =)
